.vscode/settings.json

```diff
@@ -1,3 +1,2 @@
 {
-    "editor.formatOnSave": true
```

Comment:
Oh yeah I had issues with this. Seems my local settings were not aligned, I was getting a lot of format-only diffs.

Reply:
You can disable prettier plugin for workspace